packages/dui3/lib/bindings/definitions/IAccountBinding.ts

```diff
@@ -0,0 +1,29 @@
+import type { IBinding, IBindingSharedEvents } from 'lib/bindings/definitions/IBinding'
+
+export const IAccountBindingKey = 'accountsBinding'
```

Comment:
Even in .NET world the I prefix is only applicable to interfaces - so types, not actual constants with values in them. This will mislead people reading this import in other files.